roles/openshift_version/tasks/set_version_containerized.yml

```diff
@@ -2,6 +2,21 @@
 - set_fact:
     l_use_crio: "{{ openshift_use_crio | default(false) }}"
```

Comment:
Shouldn't this move to the `defaults/main.yml`.

Probably should happen in another PR.